### DISCUSSION TITLE
New way to add service

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/edit.tosdr.org#990] [Ideas]
* fill in the domain name (could even be from the browser extension!)
* if no service with this domain name exists yet, one is created.
* name gets set to domain name without the extension.
* its main page gets crawled and anchor texts get listed. you can tick checkboxes for documents that you think need to be reviewed.
* same for links in those docs.

Schema change needed: document needs a list of anchor texts to get to it from the main page.

Next step: crawling for tags can still be done in Ruby, but crawling the docs needs to be done by ambanum/CGUs.

**Comment by nm17** [comment]
I was about to make the same issue. There needs to be some way to add services.

**Comment by JustinBack** [comment]
There is! https://forum.tosdr.org/request
